tomerlichtash/mels-loop
# infra(optimize): Bump dependencies and optimize CI

- test-e2e
- bump
- yarn.lock
- bump
- bump
- downgrade axiom
- downgrade axiom
- bump stylable
- bump next
- bump pw
- remove prettier
- bump config.yml
- bump with reports
- store_test_results
- should fail
- use junit for vitest
- use node 18
- fix pw and node
